MANIFEST.in

```diff
@@ -2,20 +2,23 @@
 include CHANGES COPYING COPYING.LESSER
 
 # Files from setup.py package_data that are not automatically added to source distributions
-recursive-include lib/iris/tests/results *.cml *.cdl *.txt *.xml *.json
+recursive-exclude lib/iris/tests *
+recursive-exclude lib/iris/fileformats/_pyke_rules/compiled_krb *
 recursive-include lib/iris/etc *
-include lib/iris/fileformats/_pyke_rules/*.k?b
-include lib/iris/tests/stock*.npz
+include lib/iris/fileformats/_pyke_rules/*.krb
 
-include requirements/*.txt
+recursive-include requirements/ci *
```

Comment:
The syntax is `recursive-include <dir> <glob>` rather than `recursive-include <dir><glob>`... but yeah, I know what you mean. I'll pull it up one directory :+1:

Reply:
Done :+1: